### ISSUE TITLE
❗Repo not found or no token provided

### ISSUE THREAD

**Issue by rommansabbir** [DImuthuUpe/AndroidPdfViewer#1189]
Issue: 
```
Could not HEAD 'https://jitpack.io/com/github/barteksc/android-pdf-viewer/2.8.2/android-pdf-viewer-2.8.2.pom'. Received status code 401 from server: Unauthorized
```

Repository:
```
maven(url = "https://jitpack.io")
```

**Comment by rommansabbir** [comment]
Screenshot from `JitPack.io`

![Screenshot 2024-06-23 at 1 09 24 PM](https://github.com/DImuthuUpe/AndroidPdfViewer/assets/25950083/060cb2c9-f109-4eac-805a-0d8798bbf245)

**Comment by DImuthuUpe** [comment]
Use this one instead https://www.jitpack.io/#DImuthuUpe/AndroidPdfViewer

**Comment by zack-barakat** [comment]
@DImuthuUpe version 2.8.2 is not available there, only 2.8.1.
Can you please include 2.8.2 or if possible provide the release note for the removed 2.8.2 so we know what features and fixes we are missing by downgrading to 2.8.1
